fpdf/ttfonts.py

```diff
@@ -414,7 +418,7 @@ def extractInfo(self):
         # hmtx - Horizontal metrics table
         self.getHMTX(numberOfHMetrics, numGlyphs, glyphToChar, scale)
 
-    def makeSubset(self, file, subset):
+    def makeSubset(self, file, subset, remap = {}):
```

Comment:
Could you maybe add a short docstring explaining what is the role of this `remap` dict,
and what are its keys & values?

Also, please do not set a mutable default value as it can be very error prone:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html